packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -104,6 +105,17 @@ export const Feature2TemplateName: any = {
 
 const workflowFileName = "app.yml";
 
+const M365Actions = [
+  "botAadApp/create",
+  "teamsApp/create",
+  "teamsApp/update",
+  "aadApp/create",
+  "aadApp/update",
+  "m365Bot/create",
+  "m365Bot/update",
+];
+const AzureActions = ["arm/deploy"];
```

Comment:
Could we add optional properties in action interface? Coordinator can use the properties to get the action type.
If so, we will not modify the coordinator code when the action changes

Reply:
Can do it, but not now.